### DISCUSSION TITLE
STDF-Viewer only shows trend/histogram/statistics from one part

### DISCUSSION THREAD

**Discussion by POShea12** [noonchen/STDF-Viewer#147] [Q&A]
I'm trying out v4.0.0 on Windows 10.   I downloaded the zip file, and ran the application from the unzipped directory.

I can open a stdf file and it seems to import the data correctly (debug file appended).  

When I highlight a test in the left pane, the DUT Summary does show all the 4 parts in the stdf file I've opened.  The file is stdf version 4.  However, the Trend Chart, Histogram and Test Statistics only show the data from the first part.

Am I somehow using this incorrectly?  How do I see data from all parts?

[stdf_debug.txt](https://github.com/noonchen/STDF-Viewer/files/11389708/stdf_debug.txt)

**Comment by noonchen-ni** [comment]
Hi @POShea12 ,

Sorry for the late reply, I didn't check github notifications for months...

As for the issue you're facing, the only reason I can think of is that the test value is NAN or INF, in such case I deliberately discard the value from the chart.

You can use STDF to XLSX tool under `Utilities` to see the actual field values of each record in your stdf file.
